packages/venia-ui/lib/components/CheckoutPage/OrderConfirmationPage/orderConfirmationPage.js

```diff
@@ -51,6 +51,7 @@ const OrderConfirmationPage = props => {
             firstname={firstname}
             lastname={lastname}
             email={email}
+            isCancelButtonHidden={true}
```

Comment:
Since this is the default we don't need to provide it, right?

Reply:
Yeah, it is the default, but I just wanted to provide it for aesthetic reasons? Is that uncommon in our code base?

Reply:
I think you'll probably find a mix. If you'd like to keep it, that's fine :)

Reply:
Removed the props in this component because it is (I just realized) pointing to a different version of the create account component that is only used in the order confirmation page.

On a different topic, do you know why we had to create a new component for create account section on the order confirmation page instead of utilizing the old one?

Reply:
> do you know why we had to create a new component for create account section on the order confirmation page instead of utilizing the old one

Probably because there were differences in the UI or in the logic. I believe `packages/venia-ui/lib/components/CreateAccount/createAccount.js` kicks you out to the home page once signed in whereas `packages/venia-ui/lib/components/CheckoutPage/OrderConfirmationPage/createAccount.js` does not. You could diff the components/talons to see the differences if you are curious.

We had a similar discussion yesterday in dev time about reusing the existing components for address book or creating anew and I think the outcome was that we prefer to just write new components and do the optimization of generalizing the components later.